### PR TITLE
Price table

### PR DESCRIPTION
**Pull request by velmyk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous">
+    <link rel="stylesheet" href="../price table/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
```
Comment (Nick-Shcherban) [inline index.html:9]: do you really need all these fonts?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <title>Prime table</title>
+</head>
+<body>
+    <div class="section_1">
+        <h1 class="heading_1">Basic <div class="subtitle">Website</div></h1>
+        <div class="cost_1">
+            <div class="cost">99€</div>
+        </div>
+        <p class="paragraph">Simple website made in the world's most populars CMS system, Wordpress.</з>
```
Comment (Nick-Shcherban) [inline index.html:18]: create understandable names of classes

File: style.css [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    height: 770px;
+    overflow: hidden;
+}
+.Popular-background {
+    position: absolute;
+    width: 25.01px;
+    height: 147.08px;
+    background-color: #dc3f45;
+    position: absolute;
+    top: -38.41px;
```
Comment (Nick-Shcherban) [inline style.css:36]: do you really need negative positioning?

**Review by Nick-Shcherban — COMMENTED** [review]
pay attention to class names. Try to give more descriptional names.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous">
+    <link rel="stylesheet" href="../price table/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>Prime table</title>
```
Comment (Nazar0n) [inline index.html:10]: Price*

File: index.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+</head>
+<body>
+    <div class="section_1">
+        <h1 class="heading_1">Basic <div class="subtitle">Website</div></h1>
+        <div class="cost_1">
+            <div class="cost">99€</div>
+        </div>
+        <p class="paragraph">Simple website made in the world's most populars CMS system, Wordpress.</з>
+        <ul class="list_1" >
+            <li class="list-item"><img class="shape1" src="../price table/Shape1.png" alt="shape1" width="15" height="13"><div class="text"> Website in WordPress</div></li>
```
Comment (Nazar0n) [inline index.html:20]: fix your `src` for pics

File: style.css [hunk 6]
```diff
@@ -0,0 +1,252 @@
+body {
+    display: flex;
+    position: relative;
+    width: 1440px;
+    height: 1000px;
+    background-color: #141b1c;
+}
+.section_1 {
+    position: absolute;
```
Comment (Nazar0n) [inline style.css:9]: you don't need `position: absolute`?

File: index.html (hunk 5 above)
Comment (Nazar0n) [inline index.html:20]: don't need `space` in file names

File: index.html [hunk 7]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous">
+    <link rel="stylesheet" href="../price table/style.css">
```
Comment (Nazar0n) [inline index.html:7]: fix your `href`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>Prime table</title>
+</head>
+<body>
+    <div class="section_1">
+        <h1 class="heading_1">Basic <div class="subtitle">Website</div></h1>
```
Comment (Nazar0n) [inline index.html:14]: use `span` instead of `div`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        <div class="cost_1">
+            <div class="cost">99€</div>
+        </div>
+        <p class="paragraph">Simple website made in the world's most populars CMS system, Wordpress.</з>
+        <ul class="list_1" >
+            <li class="list-item"><img class="shape1" src="../price table/Shape1.png" alt="shape1" width="15" height="13"><div class="text"> Website in WordPress</div></li>
+            <li class="list-item"><img class="shape2" src="../price table/Shape1.png" alt="shape1" width="15" height="13"><div class="text"> 3 Sub pages</div></li>
+            <li class="list-item"><img class="shape3" src="../price table/Shape1.png" alt="shape1" width="15" height="13"><div class="text"> Themes customized</div></li>
```
Comment (Nazar0n) [inline index.html:22]: format document with prettier

File: index.html [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+            <li class="list-item"><img class="shape12" src="../price table/Shape2.png" alt="Shape2" width="15" height="13"><div class="text"> 1 month free support</div></li>
+        </ul>
+        <button class="button_2">Order now</button>
+    </div>
+    <div class="section_3">
+        <h1 class="heading_3">Webshop <div class="subtitle">Website</div></h1>
+        <div class="cost_3"><div class="cost">599€</div></div>
+        <p class="paragraph">Your very own webshop made the world's mosy popular CMS system, Wordpress</p>
+        <ul class="list_3 ">
+            <li class="list-item"><img class="shape13" src="../price table/Shape3.png" alt="shape3" width="15" height="13"><div class="text"> Webshop in WordPress</div></li>
```
Comment (Nazar0n) [inline index.html:49]: why do you need uniqes classes for shapes

File: index.html [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+            <li class="list-item"><img class="shape14" src="../price table/Shape3.png" alt="shape3" width="15" height="13"><div class="text"> 3-5 sub pages</div></li>
```
Comment (Nazar0n) [inline index.html:50]: remove unused classes for elements

File: style.css [hunk 12]
```diff
@@ -0,0 +1,252 @@
...
+.Popular {
+    position: absolute;
+    width: 75.76px;
+    height: 17px;
+    left: -23.93px;
+    right: 6.02px;
+    top: 67.31px;
+    bottom: 28.79px;
+    font-family: 'Montserrat';
+    font-style: normal;
```
Comment (Nazar0n) [inline style.css:52]: do you need `font-style: normal`?

File: style.css [hunk 13]
```diff
@@ -0,0 +1,252 @@
...
+    background-color: #334247;
+    width: 380px;
+    height: 770px;
+}
+h1 {
+    position: absolute;
+    color: #ffffff;
+    font-size: 35px;
+    text-align: center;
+    font-family: 'Montserrat';
```
Comment (Nazar0n) [inline style.css:75]: use `font-family` in `body` if you have 1 font for all page

File: style.css [hunk 14]
```diff
@@ -0,0 +1,252 @@
...
+    position: absolute;
+    top: -38.41px;
+    right: 21.69px;
+    bottom: 658.91px;
+    left: 332px;
+    transform: rotate(-45deg);
+    
+}
+.Popular {
+    position: absolute;
```
Comment (Nazar0n) [inline style.css:44]: use `flex-box` for page

**Review by Nazar0n — CHANGES_REQUESTED** [review]
Learn html

File: style.css (hunk 13 above)
Comment (20Sasha20) [reply style.css:75]: the code uses 2 types of fonts

File: style.css (hunk 14 above)
Comment (20Sasha20) [reply style.css:44]: flex is specified in the body

File: index.html (hunk 9 above)
Comment (20Sasha20) [reply index.html:22]: I don't quite understand the comment

File: style.css (hunk 3 above)
Comment (20Sasha20) [reply style.css:36]: without such positioning, the element will not be in the right place

Comment (Nazar0n) [reply style.css:36]: use flexbox

File: index.html (hunk 9 above)
Comment (Nazar0n) [reply index.html:22]: Відформатуй файл за допомогою prettier

**Pull request merged**